lib/util/url.spec.ts

```diff
@@ -10,220 +11,190 @@ import {
   parseUrl,
   replaceUrlPath,
   resolveBaseUrl,
+  trimLeadingSlash,
   trimSlashes,
   trimTrailingSlash,
 } from './url';
 
 describe('util/url', () => {
-  it.each`
```

Comment:
Why do you remove all those tests? they are unrelated to your changes, so leave them as is. otherwise it's more complicated to see unwanted side effects from your changes.